README.md

```diff
@@ -1 +1,45 @@
-﻿# Todo List App
+
+# AwesomeBooks
+
+This is a wesbite where you can add and remove books.
+
+![screenshot](Screenshottt.png)
+
+This project is my portfolio.
+
+## Built With
+
+- Javascript
+- HTML
+- CSS
+- Wabpack
+
+[Live Demo](https://github.com/zairdon20/Todo.git)
+
+## Getting Started
+
+### install
+
+- run npm start to view the project locally.
+
+To get the content of this project locally you need to run this command in your terminal:
+
+- git clone your https://github.com/zairdon20/Todo.git
+- cd project My-Portfolio
+- open the index.html file to see project running.
+
+### Deployment
+
+This project is deployed on a github page on the link below:
+
+- https://zairdon20.github.io/Todo/
+
+👤 **Josphat Nkonde**
+
+- GitHub: [@zairdon20](https://github.com/zairdon20)
+- Twitter: [@Josphat1](https://twitter.com/Josphat1/)
+- LinkedIn: [zairdon](https://www.linkedin.com/in/zairdon/)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] when including the link to the license please make sure to add the file to your repository in order to make it work. Have a look at [the template](https://github.com/microverseinc/readme-template) provided by Microverse for an example.